agr_literature_service/api/tests/test_013_reference_cont.py

```diff
@@ -34,25 +37,100 @@ def test_reference_create_with_existing_items():
         ],
         "resource": 'AGR:AGR-Resource-0000000001',
         "title": "Another title",
-        "volume": "433",
+        "volume": "013a",
         "open_access": True
     }
-    assert 1 == 1
-    # process the reference.
-    reference = ReferenceSchemaPost(**full_xml)
-    res = create(db, reference)
-    assert res == 'AGR:AGR-Reference-0000000005'
+    # process the references
+    ref1 = ReferenceSchemaPost(**full_xml)
+    print(ref1)
+    res1 = create(db, ref1)
+    # res1 = router_create(ref1)
+    bob = show(db, res1)
```

Comment:
bob variable, probably ok

Reply:
If you ever see bob or variations of it in my code then it probably means it was supposed to be temporary and removed on the tidy at the end.

Reply:
cool cool.  I saw there were BOB strings earlier and were gone by this point, so wasn't sure if the bob variable was final.  Also wasn't sure if the PR was for final, or just for sanity check.